src/main/java/gov/cabinetoffice/gapfindapiadmin/services/ApiGatewayService.java

```diff
@@ -28,6 +29,8 @@ public class ApiGatewayService {
 
     private final ApiKeyService apiKeyService;
 
+    private final ApiKeyController apiKeyController;
```

Comment:
We should never be injecting controllers into service classes. If the method you need to use lives in a controller but is not strictly dedicated to handling web requests then this suggests a design problem and that the method should be moved out of the controller.